addons/cargo/initSettings.sqf

```diff
@@ -42,3 +42,12 @@
     false,
     {[QGVAR(enableRename), _this, true] call EFUNC(common,cbaSettings_settingChanged)}
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(carryAfterUnload), "CHECKBOX",
+    [LSTRING(carryAfterUnload), LSTRING(carryAfterUnload_description)],
+    [LELSTRING(OptionsMenu,CategoryLogistics), LLSTRING(openMenu)],
+    true,
+    true,
```

Comment:
I don't think this setting has to be global, it's more based on individual preference.

Reply:
There are many personal pref settings already. Just best to allow it be decided on all 3 levels, it's their problem.

Reply:
Yep, bad copy paste on my end, will amend :)